packages/toolpad-app/next.config.mjs

```diff
@@ -57,19 +57,46 @@ const regexEqual = (x, y) => {
   );
 };
 
+const securityHeaders = [
+  {
+    key: 'Strict-Transport-Security',
```

Comment:
this one needs to get optional for self hosters

Reply:
in fact I'd even argue the responsibility for this one doesn't belong in the application. I'm on my phone so i can't verify, but I'd be surprised if render.com doesn't already set it.

Reply:
Ok, I'm removing it for this PR, we can move more progressively on this header.

Reply:
> but I'd be surprised if render.com doesn't already set it.

They don't https://toolpad-gfpr.onrender.com/. But their certificate is issued by Cloudflare while on https://master--toolpad.mui.com/ the certificate is issued by MUI